rest-api/src/main/java/gov/cms/qpp/conversion/api/model/UnprocessedCpcFileData.java

```diff
@@ -7,11 +7,13 @@
  * Model to hold converted metadata to Unprocessed Cpc file data.
  */
 public class UnprocessedCpcFileData {
-	private String fileId;
-	private String filename;
-	private String apm;
-	private Instant conversionDate;
-	private Boolean validationSuccess;
+
+	private final String fileId;
+	private final String filename;
+	private final String apm;
+	private final Instant conversionDate;
+	private final Boolean validationSuccess;
+	private final String purpose;
```

Comment:
Other than telling the CPC+ team that this `UnprocessedCpcFileData` came from some purpose, what about this other idea?  Allow a header to come in for the `/cpc/unprocessed-files` URL (`CpcFileControllerV1#getUnprocessedCpcPlusFiles`).  If that header specifies "test" as the purpose, only return the `Metadata`s that has the same purpose.  That way when the CPC+ team calls the API, they will not specify "test" for the header, and they'll only get the real data.  But Jeff Benton will call this API with "test", and he'll only ever get the test files.

What do you think?

Reply:
These changes allow us to do more than that, though.  This can also help us generate some stats for our Splunk dashboard and will help out the A&R team to provide reports.

Reply:
At any rate... i wouldn't do that with a header.  I'd rather it be done something like `/cpc/unprocessed-files/{purpose}` or `/cpc/unprocessed-files?purpose={purpose}`.